compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_infinite_loop_around_core_block.mlir

```diff
@@ -0,0 +1,238 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-amdaie-insert-infinite-loop-around-core-block),canonicalize)" --split-input-file %s | FileCheck %s
```

Comment:
Is it necessary to canonicalize in this lit test?

Reply:
Yes, leads to better and simpler code checks.